data/hai/unfettered 1 alphas.txt

```diff
@@ -0,0 +1,1532 @@
+	to complete
+		has "<uhai> 9: Alpha Base: offered"
+		has "flagship base attribute: alpha logs"
+	on complete
+		outfit "Jump Drive" -10
+		log `Captured the Alpha Auxiliary. The Alphas had a trap set but it failed, and I was able to rescue a few Korath prisoners. Or, at least, that's what the Alphas want everyone to think.`
+		payment 10000000
+		"salary: Unfettered" = 1000
+		event "unfettered: korath back up" 20 50
+		conversation
+			`As you land with the Auxiliary, you notice dozens of Unfettered start coming your way, cheering, and their numbers quickly grow to hundreds. More keep coming from every direction, probably expecting Jump Drives, but this could also be the first time they see the fabled Auxiliary in person. Whatever the case, they seem surprised when they realize it belongs to you. They murmur with each other and then they start saluting, by rising one fist up. First a few do it, then more, until finally all of them are saluting in respect.`
+			choice
+				`	(Raise my fist like warriors do.)`
+				`	(Bow.)`
+					goto bow
+				`	(Do nothing.)`
+					goto stadium
+			`	They approve of your risen fist, and seem happy you understand their culture.`
+				goto stadium
+			label bow
+			`	It does not seem to be the right reaction at all. They shout at you; all form of respect is gone. One very shocked Unfettered soldier even tries to shoot you for this insult! It looks like you have shown yourself to be weak to them. After some time, a warlord arrives and calms them down by rising their blade, that shines brightly enough to draw their attention.`
+			label stadium
+			`	The crowd leads you to the stadium.`
+			`	The council seems surprised to see you return, especially the Unfettered warlord that annoyed you earlier, but they address you with a deeper respect than before. "You have proven yourself. Now we have almost enough jump drives for the next phase..." You look at them intensely, and after a few moments they realize, "Oh right, your payment! Here it is: <payment>! From now on, you will be referred to as Herald."`
+			choice
+				`	"I have found Korath prisoners on the Alpha ship."`
+			`	This seems to trouble them deeply. They are split between those who want to interrogate the Korath for their technological secrets, and those who still remember too freshly the horrors of the Korath raids. The Chiefs are now the most active ones - some show scars or speak of lost family, others insult the Korath. A few, however, keep their calm.`
+			`	Then, finally, one of them speaks up. He had remained quiet through the argument, probably thinking. You see many scars on his arms as he stands, and can only guess at what he has been through. He speaks calmly, yet firmly, strongly enough to make everyone listen.`
+			`	You recognize Choot'k, he looks different without his armor but there can be no doubts that it's him.`
```

Comment:
```suggestion
			`	You recognize Choot'k. He looks much different without his armor but there can be no doubt that it's him.`
```

Reply:
much different, is that correct?

Reply:
I think? If there’s a better synonym, use that.

Reply:
Resolved in 3800bed